### PR TITLE
Important Update

### PR DESCRIPTION
**Pull request by james-hollinger**
💯💯💯💯💯💯💯💯💯💯

### CODE REVIEW

File: documentation/Architecture.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 PRIME is a traditional N-Tier application architecture. It's composed of a frontend Angular application that communicates to a RESTful .Net Core API which persists it's data in a Postgres Database. We use Keycloak to authenticate users against BCSC and SiteMinder IDIR. There is one external integration with a HIBC REST API that validates College Licences against our users.
 
 ## Technologies in use
-- .Net Core 3.1 C# 
+- .Net Core 3.1 C#
```
Comment (neophyte57) [inline documentation/Architecture.md:5]: Congratulations on the 1000th pull request!

**Review by neophyte57 — APPROVED** [review]

**Comment by james-hollinger** [comment]
🎊🎊🎊🎊

**Review by AnaisHebert — APPROVED** [review]
🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉
Does this mean we're done?😁

**Review by mtpultz — APPROVED** [review]
image

**Comment by james-hollinger** [comment]
> 
> 
> 🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉
> Does this mean we're done?😁

I think it does.

File: documentation/Architecture.md (hunk 1 above)
Comment (itchison) [inline documentation/Architecture.md:5]: If we're stating the version of .net we should probably also state the version of c#

**Review by itchison — DISMISSED** [review]

**Review by hogmann — APPROVED** [review]

**Review by johnnyl6 — APPROVED** [review]

File: documentation/Architecture.md (hunk 1 above)
Comment (mtpultz) [reply documentation/Architecture.md:5]: Feels like this is out of scope we want to keep this MVP

**Review by joliesummers — APPROVED** [review]
YAY!

File: documentation/Architecture.md (hunk 1 above)
Comment (mtpultz) [reply documentation/Architecture.md:5]: @joliesummers can you make a second ticket to add the language version to the docs image

**Comment by ajdeziel** [comment]
Chris Pratt is wowed by this 1000th PR!

**Review by ajdeziel — APPROVED** [review]
Chris Pratt is **WOWED** by this 1000th PR, as am I!

**Pull request merged**